docs/loading_data.md

```diff
@@ -18,6 +18,12 @@ docker exec -it student_dashboard /bin/bash -c \
 After about 30 to 60 seconds, the cron job should have completed and you should have data!
 In the admin interface, there is a table where you can check the status of the cron job runs.
 
+Debugging cron use the following command. Once you run this command then go to VSCode and start the 'MyLA Docker Cron'. Add some 
+Breakpoint to debug at desired location.
+```sh
+docker exec -it student_dashboard /bin/bash -c "DEBUGPY_WAIT_FOR_ATTACH=True DEBUGPY_ENABLE=TRUE DEBUGPY_REMOTE_PORT=3001 ./manage_debugpy.py runcrons --force"
+```
```

Comment:
The documentation was missing for debugging cron, This was Quiet useful when resolving Cron issues

Reply:
This command is already included in https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/docs/CONTRIBUTING.md

Reply:
Removed it from here https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1554/commits/ca01926cd8de9537ed3c7edf31d2288e20e641f6